HW_17/utilities/config_reader.py

```diff
@@ -0,0 +1,22 @@
+import configparser
+from constants import ROOT_DIR
+
+
+abs_path = f'{ROOT_DIR}/configurations/configuration.ini'
+config = configparser.RawConfigParser()
+config.read(abs_path)
+
+
+def get_application_url():
+    return config.get('app_data', 'app_url')
+
+
+def get_user_creds():
+    return (config.get('user_data', 'email'),
+            config.get('user_data', 'password'))
+
+
+def get_browser_id():
+    return config.get('browser_data', 'browser_id')
+
+
```

Comment:
please split tests to different test suites(test_login_page.py, test_mail_page.py ......)